cfme/tests/cloud_infra_common/test_vm_instance_analysis.py

```diff
@@ -303,10 +288,16 @@ def get_ip_address():
 
 
 @pytest.fixture(scope="module")
-def ssa_vm(ssa_single_vm, assign_profile_to_vm):
+def ssa_vm(ssa_single_vm, assign_profile_to_vm, ssa_provider, request, appliance):
     """Single vm with assigned profile"""
     ssa_vm = ssa_single_vm()
     assign_profile_to_vm(ssa_vm)
+    if ssa_provider.one_of(VMwareProvider):
+        set_host_creds(request, appliance, ssa_provider, ssa_vm)
+
+    if ssa_provider.one_of(EC2Provider):
```

Comment:
Required:
It should `elif`

Question:
what about other providers? If the provider does not belong to `VSphere` or `EC2`

Reply:
It is only required for EC2 provider.

Reply:
@digitronik I will try to write or update host creds for RHV provider as well.

Reply:
Credentials are added for all Infra providers. :)